### PR TITLE
use the raw issue dict for export, print and search

### LINKED ISSUE #2: template for jira issues/jira api fields
add a template or config file in which the user can set which fields of the jira tickets shall be written, the config file must contain information on how the data needs to be added to a jira issue on the server

### PR DESCRIPTION
**Pull request by Timuuc**
remove unused constants

close #2

### CODE REVIEW

File: src/pyJiraCli/issue_constants.py [hunk 1]
```diff
@@ -47,14 +47,6 @@
     '8' : 'QMeldung'
 }
 
-ISSUE_PRIORITIES = {
```
Comment (gabryelreyes) [inline src/pyJiraCli/issue_constants.py:50]: Are any other constants required? Can we remove the whole file?

File: src/pyJiraCli/jira_issue.py
Comment (gabryelreyes) [inline src/pyJiraCli/jira_issue.py:1]: This is a wrapper of the Jira.Issue class of the Jira Library. Can we remove this file completely?

**Review by gabryelreyes — CHANGES_REQUESTED** [review]

**Comment by gabryelreyes** [comment]
No new changes? @Timuuc

File: src/pyJiraCli/issue_constants.py (hunk 1 above)
Comment (Timuuc) [reply src/pyJiraCli/issue_constants.py:50]: The import function still uses the issue constants. This should be addressed in a seperate issue

**Pull request merged**